.github/workflows/release.yml

```diff
@@ -69,7 +87,32 @@ jobs:
           TAG_NAME: ${{steps.semantic-release.outputs.TAG_NAME}}
           RELEASE_BODY: ${{github.event.inputs.change-log}}
           REPO_NAME: twilio/twilio-cli
+  delete-draft-release:
```

Comment:
This is cleanup step to:
1. Delete the tagged (draft) release.
2. Revert the last commit made by semantic-bot, which has changelog.